calendville/tests/test_list_appointments.py

```diff
@@ -41,14 +41,15 @@ def test_appointment_exist(self, setup, logged_user):
         appointments_list = response.context["appointments_week"]
         assert len(appointments_list[0]["appointments"]) > 0
 
-    # TODO: Skipping since this test file is pending to review by its owner
-    @pytest.mark.skip
     def test_appointment_do_not_exist(self, setup, logged_user):
         client, user = logged_user()
         path = reverse("list_appointments")
         response = client.get(path)
         appointments_list = response.context["appointments_week"]
-        assert len(appointments_list[1]["appointments"]) > 0
+        number_appointments = 0
+        for i in range(0, 6):
+            number_appointments += len(appointments_list[i]["appointments"])
```

Comment:
Se puede reducir a
`number_appointments = sum([len(day) for day in appointments_list])`